packages/file_selector/file_selector_ios/ios/file_selector_ios.podspec

```diff
@@ -0,0 +1,24 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint file_selector_ios.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'file_selector_ios'
+  s.version          = '0.0.1'
+  s.summary          = 'iOS implementation of file_selector.'
+  s.description      = <<-DESC
+Displays the native iOS document picker.
+                       DESC
+  s.homepage         = 'https://github.com/flutter/plugins/tree/main/packages/file_selector'
+  s.license          = { :type => 'BSD', :file => '../LICENSE' }
+  s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
+  s.source           = { :http => 'https://github.com/flutter/plugins/tree/main/packages/file_selector/file_selector_ios' }
+  s.source_files = 'Classes/**/*'
```

Comment:
I see the warning
```
no rule to process file '/Users/m/Projects/plugins/packages/file_selector/file_selector_ios/ios/Classes/FileSelectorPlugin.modulemap' of type 'sourcecode.module-map' for architecture 'arm64'
```
Because a `module_map` is specified this needs to be:
```suggestion
  s.source_files = 'Classes/**/*.{h,m}'
```
or the modulemap will try and fail to be compiled into the framework.


<img width="545" alt="Screen Shot 2022-08-19 at 5 36 21 PM" src="https://user-images.githubusercontent.com/682784/185722496-dc9239cc-f5c0-40af-8380-1f55926adad2.png">

Another package that does this:
https://github.com/flutter/plugins/blob/f48198e40ee3bc2b4390f0a8b86c448c5444f616/packages/webview_flutter/webview_flutter_wkwebview/ios/webview_flutter_wkwebview.podspec#L17

Reply:
Fixed.